windows/windows_update/win_update_restore_snapshot.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   This case is to restore VM to the original base snapshot, which is not installed with the msu file
+#   by playbook windows_update.yml
+#
+- name: win_update_restore_snapshot
+  hosts: localhost
+  gather_facts: no
+  vars_files:
+    - "{{ testing_vars_file | default('../../vars/test.yml') }}"
```

Comment:
No vars_files in test cases any more. The vars file has been included in main.yml. So please remove line 11~12.